library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -492,6 +495,21 @@ void vtkF3DRenderer::ConfigureGridUsingCurrentActors()
   this->ResetCameraClippingRange();
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::ShowHDRISkybox(bool show)
```

Comment:
Can we have a hotkey to toggle it? That would be awesome!

Reply:
that would be trivial, but I need to improve perf of HDRI, this will come in the next PR

Reply:
ok